components/02-elements/dropdown-list/dropdown-list.js

```diff
@@ -1,18 +1,12 @@
 'use-strict';
 
-(function() { // eslint-disable-line
+(function DropdownList() {
   const dropdownCollapseLabel = '.dropdown-list__item--collapse .dropdown-list__label',
         dropdownItem          = [ ...document.querySelectorAll(dropdownCollapseLabel) ],
```

Comment:
I know it's old code, but should we change `dropdownItem` to `dropdownItems` here ? 
and in line 76 and 77 `dropdownItems` to `dropdownMediumOpenItems` ?

Reply:
From my point of view, I would change the whole structure of this file so lets leave it this way.

Reply:
@kruchy8 added #267 https://github.com/SnowdogApps/magento2-alpaca-components/issues/267 - you can add your point of view there (not to be lost)